holmes/common/env_vars.py

```
@@ -22,7 +22,7 @@ def load_bool(env_var, default: bool):
 STORE_EMAIL = os.environ.get("STORE_EMAIL", "")
 STORE_PASSWORD = os.environ.get("STORE_PASSWORD", "")
 HOLMES_POST_PROCESSING_PROMPT = os.environ.get("HOLMES_POST_PROCESSING_PROMPT", "")
-ROBUSTA_AI = load_bool("ROBUSTA_AI", False)
+ROBUSTA_AI = load_bool("ROBUSTA_AI", True)
```

Ruff (0.8.2)

Boolean positional value in function call

(FBT003)